aws/automate_manager.py

```diff
@@ -126,7 +126,7 @@ def submit(self, mdf_rec, organization,
             }
         }
         print("Flow is ", self.flow)
-        flow_run = self.flow.run_flow(automate_rec, monitor_by=[submitting_user_id])
+        flow_run = self.flow.run_flow(automate_rec, monitor_by=[submitting_user_id, "5fc63928-3752-11e8-9c6f-0e00fd09bf20"])
```

Comment:
Would be better not to have an MDF specific magic uuid here in this general purpose class. Perhaps make a new parameter to `submit` method that has the list of monitor_by values. We could move this magic UUID up into submit_dataset.

Make sure the magic uuid is well documented there

Reply:
Just made a commit with the update.